Kiosk/App/AppSetup.swift

```diff
@@ -30,6 +30,6 @@ class AppSetup {
 
     var needsZipCode: Bool {
         // If we're swiping with the card reaer, we don't need to collect a zip code.
-        return disableCardReader
+        return false
```

Comment:
OK cool – I'll leave it up to you to open an issue on this repo (mark it as a "chore").